docs/fees_and_rewards/Dynamic_Issuance.md

```diff
@@ -0,0 +1,277 @@
+</Collapsible>
+
+`block_reward(block_height) → Balance`
+
+The reward to be issued to the proposer for this block is defined by a reward function and computed for each block using latest `AvgBlockspaceUsage`, `transaction_byte_fee` and `block_height`.
+
+
+```rust
+reference_subsidy-blockspace_utilization*min(reference_subsidy,max_block_fee)
+```
+where
+
+```rust
+reference_subsidy = reference_subsidy_for_block(ProposerSubsidyParams, block_height)
+
+max_block_fee = MAX_NORMAL_BLOCK_LENGTH * transaction_byte_fee
+
+blockspace_utilization=AvgBlockspaceUsage/MAX_NORMAL_BLOCK_LENGTH
+```
+
+<Collapsible title="Original Formula Simplification Note">
+The above definition is an approximation of the following hyperbolic formula for the chosen parameters below:
+<center>
+$a+b\tanh(-c(\text{blockspace\_utilization}-d))$
+</center>
+
+where 
+
+- $a =S_r - b*\tanh(c*d)$  the offset parameter (sets the amount of reward issued at 0 utilization to $S_r$).
+- $~~S_r~~$ is `reference_subsidy = reference_subsidy_for_block(ProposerSubsidyParams, block_height)`, a maximum amount of SSC issued at 0 utilization.
+- $b$  $= \frac{S_r-\max(S_r-\bar{F},0)}{\text{const}\tanh(c*d)}=\frac{\min(S_r,\bar{F})}{\text{const}\tanh(c*d)}$ is a linear sensitivity parameter.
+- $\bar F$ is `max_block_fee = MAX_NORMAL_BLOCK_LENGTH * transaction_byte_fee`, maximum possible amount of storage fees in this block.
+- (const) $c$ is hyperbolic sensitivity parameter (determines the shape of the reward function) (currently, 0.99)
+
+    <Collapsible title="Note">
+        c is a parameter that controls how slow will rewards go down as utilization ratio goes up. This is useful for not letting the rewards go too down when utilization ratios are small. The higher the values for $c$, the slower the rewards will decay as the utilization ratio goes up. Values near zero imply in a straight line from rewards being reference_subsidy(utilization=0) until zero (utilization=1). Values around ~1.0 imply in a somewhat curved line from rewards being reference_subsidy(g=0) until zero (g=1). For instance, when g=0.5, the rewards would be around ~20% larger than compared with the first case. As values goes way above 1.0, then the result is to have a relatively constant reward for low values of utilization ratio, with a exponential decay afterwards. Generally speaking, Values between 0 and 1 imply that it is rational for farmers to try to maximize utilization ratio. For values above 1.0 that's not obvious, as the peak profit (rewards + fees) may be maximized on inflection point (generally between 60-80% of the utilization ratio)
+    </Collapsible> 
+- $\text{blockspace\_utilization}$ = `AvgBlockspaceUsage/MAX_NORMAL_BLOCK_LENGTH`
```

Comment:
This doesn't seem to render correctly on the frontend

Reply:
![Screenshot 2024-04-16 at 5 30 44 PM](https://github.com/subspace/protocol-specs/assets/89309559/0424130a-8271-4193-bcfc-e4f553e05bd3)

I check it out again. What is not correct here? 🤔

Reply:
Hmm looks like it's Github only issue then 
<img width="618" alt="Screenshot 2024-04-17 at 10 22 38" src="https://github.com/subspace/protocol-specs/assets/35382599/646c5952-d33a-4b3b-b4aa-f688f55c7d25">

Reply:
I changed it to this format:
blockspace_utilization = `AvgBlockspaceUsage/MAX_NORMAL_BLOCK_LENGTH`